week-1/mandatory/2-classes-db/task.md

```diff
@@ -2,11 +2,19 @@
 
 ## Submission
 
-Below you will find a set of tasks for you to complete to consolidate and extend your learning from this week.  You will find it beneficial to complete the reading tasks before attempting some of these.
+Below you will find a set of tasks for you to complete to consolidate and extend your learning from this week. You will find it beneficial to complete the reading tasks before attempting some of these.
 
 To submit this homework write the correct commands for each question here:
 
 ```sql
+1.select * from rooms where rate>=100.00;
+2.select * from reservations where extract(month from checkin_date)=extract(month from current_date) and checkout_date-checkin_date>3;
+3.select * from customers where name LIKE 'm%';
```

Comment:
Have you actually tried this query in your database - it returns no rows in mine.  Values inside quotes (e.g. 'Man' are case sensitive.  Only keywords and names are not.